src/vstart.sh

```diff
@@ -1130,6 +1135,15 @@ EOF
     fi
 }
 
+start_exporter() {
+    echo "Starting Ceph exporter daemon..."
```

Comment:
Use `debug()` helper for consistency -- it directs the output to stderr:
```suggestion
    debug echo "Starting Ceph exporter daemon..."
```